src/Switch-Thorlabs_StepperMotorMinimal/main.py

```
@@ -0,0 +1,197 @@
+
+bitness = 64 if sys.maxsize > 2**32 else 32
+kinesis_path = "C:\\Program Files\\Thorlabs\\Kinesis" # if bitness == 64 else "C:\\Program Files (x86)\\Thorlabs\\Kinesis"
+try:
+    if kinesis_path not in sys.path:
+        sys.path.insert(0, kinesis_path)
+
+    clr.AddReference("Thorlabs.MotionControl.DeviceManagerCLI")
+    clr.AddReference("Thorlabs.MotionControl.ModularRackCLI")
+
+    from Thorlabs.MotionControl import DeviceManagerCLI, ModularRackCLI
+except:
```

Ruff (0.16.1)

[error] Do not use bare `except`

(E722)